sci-mathematics/gimps/Manifest

```diff
@@ -2,4 +2,4 @@ DIST gimps-30.7.9.linux32.tar.gz 5966449 BLAKE2B b31b93417857d338e5e7454e1466bd8
 DIST gimps-30.7.9.linux64.tar.gz 7100830 BLAKE2B 8ebae083d26309f7c624e38c943e3bb0c7fe40a538840970dc222ed33ba5eaa9a5c0767b822682921eaae27f839c1eca5c0204b2c135748002e5d072744afe25 SHA512 5e6f3fbcc50b9b91fda7b286392bb17e58fd8d55a8386f27e206c63c560993da9713513d5eb4355e8e934f2e8703b39fc96f3c6f01058b729330f3526d37479b
 DIST gimps-30.8.15.linux32.tar.gz 6029035 BLAKE2B 4b84ebd89d45d5f0d0de28539ea8bc5bfb0f7383f39bb02ac00d51ba46897bba222db6215578e38fb939f541dae2569f431547caa51825dcdc6845e7e62f028b SHA512 b0a30d2d629d0f08dd2838be9dbe669c7f8f3d9f50ccfedc94c453a994cba9972440182e05d8e3b74b258e6e60cff53f6521e73a7ebdb34a45701a81d8b3f671
 DIST gimps-30.8.15.linux64.tar.gz 7211031 BLAKE2B 1d0291adb255099b9c8b44e2f650831755822ab850c2ebaa86dbe9728da619a4f9c22e3e5736a70ca3657818e2748caf32a23243347c048b9d57ecc8f7b3cf42 SHA512 6034dca7644bab85da3fdb194755a076778b37dfaa0a944ee5e64915e1196b61bd5f0a291a40118648c3a81faef3ac3026230709312d2de154260c3cf736a166
-DIST gimps-30.8.16.linux64.tar.gz 7211483 BLAKE2B 0d4b6f7c8e5eae2cf1dc64700b8274a4bbb024511fbcfb206474a889cb4bbe67c91b5be958593615b4089c37a51fd0b2e0026046edb3170986e86e126646245a SHA512 2a1c37ecffa24e5688002a3b5a8d0742b3d76d38b2e580d5e7a9684f02a8d6a8410dba2d54a825db531fed792e3c7207a32a23fd15048fb2cd93eabbb9e19b66
+DIST gimps-30.8.17.linux64.tar.gz 7222132 BLAKE2B 14e0e75fbb4784fdfe27e29535eb162071f04d63456f09bb98beb2a03af2b7d9eeac8357dc8d53e9591f9035351bedde40825b7eb8f6d85035e87b317a7aa50b SHA512 988c7a387b4d39af55bd04793bdc1c64acf67b13ab905bcc19fbd17575a7ef65d3a33ac06c2290f0457a6ebe4bf92451b43bc47d0b6d913bb03eb5f67ab823d1
```

Comment:
We usually drop and add a version in two separate ebuilds. 

I'll take care of that for this PR, but please separate it next time.